.github/workflows/elixir.yml

```diff
@@ -73,6 +73,20 @@ jobs:
           mix format --check-formatted
           mix compile --warnings-as-errors
           mix dialyzer --format github
-
+      - name: Install & run Azurite
+        run: npm install -g azurite
+      - name: Run Azurite in Background
+        shell: bash
+        run: azurite-blob &
+      - name: Create integration test containers in Azurex
+        run: |
+          mix run -e '
+            Application.put_env(:azurex, Azurex.Blob.Config, storage_account_connection_string:
+              "AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;DefaultEndpointsProtocol=http;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1"
+            )
+            Azurex.Blob.Container.create(~s{test})
+            Azurex.Blob.Container.create(~s{integrationtestingcontainer})
+            Azurex.Blob.put_blob("test_blob", "test_blob_content", "text", "test")
```

Comment:
Another option is to move this into the `test_helper` checking if `integration` is set. I think you can get the included tests like this `ExUnit.configuration()[:include]`